collector/src/main/java/org/apache/hertzbeat/collector/dispatch/unit/impl/DataSizeConvert.java

```diff
@@ -17,57 +17,26 @@
 
 package org.apache.hertzbeat.collector.dispatch.unit.impl;
 
-import java.math.BigDecimal;
-import java.math.RoundingMode;
 import lombok.extern.slf4j.Slf4j;
 import org.apache.hertzbeat.collector.dispatch.unit.DataUnit;
-import org.apache.hertzbeat.collector.dispatch.unit.UnitConvert;
 import org.springframework.stereotype.Component;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
 
 /**
  * the convert of data size
  */
 @Slf4j
```

Comment:
Maybe we can remove or add some logs? There's no log output here.

Reply:
got it